### ISSUE TITLE
Unable to get DataMap.

### ISSUE THREAD

**Issue by ldulcic** [patloew/RxWear#1]
When I try to get DataMap on Wear device like this:
```java
RxWear.Data.get()
        .compose(DataItemGetDataMap.filterByPath("/picture"))
        .subscribe(dataMap -> /* do something */)
```
I get `java.lang.IndexOutOfBoundsException: Invalid index 0, size is 0`.

I found this bug report https://code.google.com/p/android/issues/detail?id=74204 which describes this problem so I don't think this is RxWear library problem but I'm wondering is there a way to make this work?

**Comment by patloew** [comment]
Issue should be fixed in RxWear 1.2.0, could you please confirm?

**Comment by ldulcic** [comment]
Yes this works in 1.2.0, thanks for such quick response.
